### DISCUSSION TITLE
Support for S330 (T8530) Video Smart Lock

### DISCUSSION THREAD

**Discussion by paulbaird87** [fuatakgun/eufy_security#901] [General]
For some reason the Video Smart Lock (S330) has the same model identifier as the Eufy Cam 3 in the list of supported devices (see URL for exact product). 
When I install this integration the smart lock shows up as "T8530" in the devices found.  The only options I have for it are a sensor "Debug" and diagnostics "reset alarm" and "trigger alarm".
Any assistance getting this into HA would be much appreciated. Happy to be the guinea pig for this device if you need one. 

https://us.eufy.com/products/e85301y1?ref=navimenu_3_copy&fliter_category=security

**Comment by fuatakgun** [comment]
You have to work with bropat via sharing your device, integration cannot add new device support

**Reply by paulbaird87** [reply]
Where would the Thanks Fuat, just jumped on the hardware support "issues" page with him now. Cheers
